### ISSUE TITLE
Stuck on light mode

### ISSUE THREAD

**Issue by cmdshft** [nimsandu/spicetify-bloom#157]
I followed the guide and tried to apply the dark theme but it refuses to use it. It reloads spotify and it stays light themed. Any help?

https://user-images.githubusercontent.com/8571823/219299366-41e3c445-cf64-463d-ada1-2ce984e00a03.mp4

**Comment by nimsandu** [comment]
Well technically you might either using marketplace version or you have moved hear from fluent.
We recommend you to remove marketplace version and install the script version. Before that make sure to remove any existing theme files(fluent or bloom script version files or marketplace versions)
Can you kindly inform after processing above procedures @cmdshft ?

**Comment by cmdshft** [comment]
Yup, just did that. Seems to have worked fine. Sad that Marketplace is so broken it seems.

However, there seems to also be one other issue that I discovered. Instead of making a new thread, I will just add it here.

I have an issue with scrolling the sidebar playlist. It gets stuck at a certain spot and I have to unlock my scroll wheel on my G502 to free spin fast enough to "break through" whatever is holding it there.

https://user-images.githubusercontent.com/8571823/219303789-718928c0-1cd5-4cc9-ae28-97987f0ce64e.mp4

**Comment by nimsandu** [comment]
@cmdshft We have to fix that Issue. The sidebar got broken with latest version and a contributor fixed it by implementing the sidebar playlist section newly. Until we fix it please use scroll bar dragging to navigate.
Sorry for the inconvenience

**Comment by cmdshft** [comment]
I see! No problem. Will it automatically pull the fix or do I have to manually download and install it with the script again when it is done?

**Comment by nimsandu** [comment]
@cmdshft Once it fixed ill let you know. Thenjust run the script again 
Thats it

**Comment by SunsetTechuila** [comment]
with LibraryX enabled it is more noticeable and removing this rule definitely fixes the issue

**Comment by SunsetTechuila** [comment]
it is from @kyrie25 old [commit](https://github.com/nimsandu/spicetify-bloom/commit/fbb88a93bf2871eb9a7280f0cf7febdcc94295d5) and doesn't seem to be needed with the current implementation

**Comment by cmdshft** [comment]
Hey I finally got around to this. Yes, commenting that line fixes the issue.
